packages/ember-testing/lib/setup_for_testing.js

```diff
@@ -29,7 +30,7 @@ export default function setupForTesting() {
   let adapter = getAdapter();
   // if adapter is not manually set default to QUnit
   if (!adapter) {
-    setAdapter(new QUnitAdapter());
+    setAdapter((typeof self.QUnit === 'undefined') ? new Adapter() : new QUnitAdapter());
```

Comment:
Can you move this logic into a new module (I'd suggest `./adapters/default`)? Then we can simply do:

```js
import DefaultAdapter from './adapter/default';
//..snip...
setAdapter(new DefaultAdapter());
```